.gitignore

```diff
@@ -16,6 +16,8 @@ lerna-debug.log*
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
+report/
+
```

Comment:
those are coverage report and not diagnostics reports so the should probably be in 1 of the coverage sections